src/test/scala/com/softwaremill/realworld/users/UsersRepositorySpec.scala

```diff
@@ -0,0 +1,62 @@
+package com.softwaremill.realworld.users
+
+import com.softwaremill.realworld.db.{Db, DbConfig, DbMigrator}
+import com.softwaremill.realworld.utils.Pagination
+import com.softwaremill.realworld.utils.TestUtils.*
+import sttp.client3.testing.SttpBackendStub
+import sttp.client3.ziojson.*
+import sttp.client3.{HttpError, Response, ResponseException, UriContext, basicRequest}
+import sttp.tapir.EndpointOutput.StatusCode
+import sttp.tapir.server.stub.TapirStubInterpreter
+import sttp.tapir.ztapir.{RIOMonadError, ZServerEndpoint}
+import zio.test.Assertion.*
+import zio.test.{Assertion, TestAspect, TestRandom, ZIOSpecDefault, assertZIO}
+import zio.{RIO, Random, ZIO, ZLayer}
+
+import java.time.{Instant, ZonedDateTime}
+import javax.sql.DataSource
+
+object UsersRepositorySpec extends ZIOSpecDefault:
+  // TODO add user add tests
```

Comment:
Add tests when user registration logic is finished

Reply:
Tests will be added later, as we agreed to complete this PR with an endpoint that is not yet functional.